src/types/module-deployments/BundleModuleMetadata.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  JsonObject,
+  JsonProperty,
+  PropertyConvertingMode,
+} from "json2typescript";
+import CommonModuleMetadata from "./CommonModuleMetadata";
+
+@JsonObject("CollectionModuleMetadata")
+export default class BundleModuleMetadata extends CommonModuleMetadata {
+  /**
+   * The amount of royalty collected on all royalties represented as basis points.
+   * The default is 0 (no royalties).
+   *
+   * 1 basis point = 0.01%
+   *
+   * For example: if this value is 100, then the royalty is 1% of the total sales.
+   */
+  @JsonProperty(
```

Comment:
what is this black magic 🪄

Reply:
lol just handles json serialization/deserialization with support for mapping the serialized data to specific property names that might be different from the class names.

It's nice because to the caller they see `externalLink` but when we serialize to json, it'll be `external_link` (which is what opensea requires)